operators/src/processing/interpolation/mod.rs

```diff
@@ -124,8 +122,8 @@ where
     source: Q,
     params: InterpolationParams,
     tiling_specification: TilingSpecification,
-    no_data_value: P,
     interpolation: PhantomData<I>,
+    _phantom_data: PhantomData<P>,
```

Comment:
remove? Can't we bind this to the interpolation field?

Reply:
good catch. will check that,